<?xml version="1.0"?>

<launch>
	<arg name="robot" default="$(env ROBOT)"/>
  <!-- sensor message gateway node (forwards sensor messages in a desired rate) -->
  <node name="kinect_image_flip" pkg="cob_image_flip" ns="/cob_image_flip" type="kinect_image_flip" output="screen"> <!--launch-prefix="valgrind"-->
    <rosparam command="load" file="$(find cob_image_flip)/ros/launch/image_flip_params.yaml"/>
	<param name="robot" type="string" value="$(arg robot)"/>
    <remap from="pointcloud_in" to="/cam3d/rgb/points"/>
    <remap from="pointcloud_out" to="/cam3d/upright/rgb/points"/>
    <remap from="colorimage_in" to="/cam3d/rgb/image_color"/>
    <remap from="colorimage_out" to="/cam3d/upright/rgb/image_color"/>
  </node>

</launch>
